### DISCUSSION TITLE
Performance of Milvus

### DISCUSSION THREAD

**Discussion by dte324** [milvus-io/milvus#15518] [Q&A and General discussion]
I am trying to compare different querying solutions, and I was wondering if anyone had any performance metrics regarding the volume of data and speed of processing. Does anyone have any information on this? Thanks in advance!

**Comment by xiaofan-luan** [comment]
Hi @dte324  We are trying to deliver the first performance report by the release of 2.0.1.
Feel free to test on your own scenario and gives us the results
